hedera-mirror-rest/viewmodel/contractLogResultsViewModel.js

```diff
@@ -0,0 +1,52 @@
+/*-
+ * ‌
+ * Hedera Mirror Node
+ * ​
+ * Copyright (C) 2019 - 2021 Hedera Hashgraph, LLC
+ * ​
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * ‍
+ */
+
+'use strict';
+
+const EntityId = require('../entityId');
+const constants = require('../constants');
+const utils = require('../utils');
+
+/**
+ * Contract logs view model
+ */
+class ContractLogResultsViewModel {
```

Comment:
```suggestion
class ContractResultLogViewModel {
```

also update the comment

Reply:
Updated comments and name